### ISSUE TITLE
If source file not exist, don't upload translation

### ISSUE THREAD

**Issue by mamantoha** [crowdin/crowdin-cli-v1#1]
```
Uploading /home/mama/ruby/crowdin/crowdin-cli/test/ru/share/site-webscripts/org/alfresco/modules/header/sites.get.properties
error: 8: File not found

```

**Comment by mamantoha** [comment]
https://github.com/mamantoha/crowdin-cli/commit/fe44883783311578c75d9bb87391d331e9846d93

**Comment by mamantoha** [comment]
fix Issue #1

https://github.com/mamantoha/crowdin-cli/commit/8b8cb7bd9d50993585f79f87872aa98f6ae3e3fc
